### ISSUE TITLE
SonarQube install error on OCP VPC Gen2 via Toolkit Tile install

### ISSUE THREAD

**Issue by mjperrins** [ibm-garage-cloud/planning#416]
Toolkit Tile install on OCP VPC Gen2 is failing with this error 

Using Tile `1.11.0` and installed on Cluster created on VPC Gen2 the follow error occured.

```
 2020/06/20 19:50:43 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [3m20s elapsed]
 2020/06/20 19:50:53 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [3m30s elapsed]
 2020/06/20 19:51:03 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [3m40s elapsed]
 2020/06/20 19:51:13 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [3m50s elapsed]
 2020/06/20 19:51:23 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [4m0s elapsed]
 2020/06/20 19:51:33 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Still creating... [4m10s elapsed]
 2020/06/20 19:51:38 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory: Creation complete after 4m15s [id=artifactory]
 2020/06/20 19:51:39 Terraform apply | module.dev_tools_sonarqube.helm_release.sonarqube-config: Creating...
 2020/06/20 19:51:41 Terraform apply | module.dev_tools_sonarqube.helm_release.sonarqube-config: Creation complete after 3s [id=sonarqube-config]
 2020/06/20 19:51:41 Terraform apply | module.dev_tools_artifactory.null_resource.create-route[0]: Creating...
 2020/06/20 19:51:41 Terraform apply | module.dev_tools_artifactory.null_resource.create-route[0]: Provisioning with 'local-exec'...
 2020/06/20 19:51:41 Terraform apply | module.dev_tools_artifactory.null_resource.create-route[0] (local-exec): Executing: ["/bin/sh" "-c" ".terraform/modules/dev_tools_artifactory/scripts/create-artifactory-route.sh tools artifactory"]
 2020/06/20 19:51:42 Terraform apply | module.dev_tools_artifactory.null_resource.create-route[0] (local-exec): route.route.openshift.io/artifactory created
 2020/06/20 19:51:42 Terraform apply | module.dev_tools_artifactory.null_resource.create-route[0]: Creation complete after 1s [id=6351661135604994870]
 2020/06/20 19:51:43 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory-config: Creating...
 2020/06/20 19:51:45 Terraform apply | module.dev_tools_artifactory.helm_release.artifactory-config: Creation complete after 3s [id=artifactory-config]
 2020/06/20 19:51:45 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.create_service_account: Creating...
 2020/06/20 19:51:45 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.create_service_account: Provisioning with 'local-exec'...
 2020/06/20 19:51:45 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.create_service_account (local-exec): Executing: ["/bin/sh" "-c" "kubectl create serviceaccount -n tools sonarqube || exit 0"]
 2020/06/20 19:51:46 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.create_service_account (local-exec): Error from server (AlreadyExists): serviceaccounts "sonarqube" already exists
 2020/06/20 19:51:46 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.create_service_account: Creation complete after 0s [id=472502782381737833]
 2020/06/20 19:51:46 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0]: Creating...
 2020/06/20 19:51:46 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0]: Provisioning with 'local-exec'...
 2020/06/20 19:51:46 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0] (local-exec): Executing: ["/bin/sh" "-c" ".terraform/modules/dev_serviceaccount_sonarqube/scripts/add-sccs-to-user.sh tools sonarqube [\"anyuid\",\"privileged\"]"]
 2020/06/20 19:51:48 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0] (local-exec): >>> Added anyuid scc to tools:sonarqube
 2020/06/20 19:51:49 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0] (local-exec): >>> Added privileged scc to tools:sonarqube
 2020/06/20 19:51:49 Terraform apply | module.dev_serviceaccount_sonarqube.null_resource.add_ssc_openshift[0]: Creation complete after 4s [id=927674383746364369]
 2020/06/20 19:51:49 Terraform apply | 
 2020/06/20 19:51:49 Terraform apply | Error: Error running command 'kubectl rollout status deployment/sonarqube-sonarqube -n tools': exit status 1. Output: Waiting for deployment "sonarqube-sonarqube" rollout to finish: 0 out of 1 new replicas have been updated...
 2020/06/20 19:51:49 Terraform apply | error: deployment "sonarqube-sonarqube" exceeded its progress deadline
 2020/06/20 19:51:49 Terraform apply | 
 2020/06/20 19:51:49 Terraform apply | 
 2020/06/20 19:51:49 Terraform apply | 
 2020/06/20 19:51:49 Terraform APPLY error: Terraform APPLY errorexit status 1
 2020/06/20 19:51:49 Could not execute action
 2020/06/20 19:51:52 Git token not available for the workspace
```

**Comment by seansund** [comment]
Duplicate of #403

**Comment by seansund** [comment]
You can update the version of the cli-tools image in `launch.sh` of the izero repository to (v0.3.0 or v0.3.1-lite). It has been released with v1.7.1
